layout/_partial/comments.ejs

```diff
@@ -40,5 +40,25 @@
                 document.getElementById("gitment-container").style.display = "block";
             </script>
         <% } %>
+
+
+    <% } else if (theme.comments.gitalk.clientID && theme.comments.gitalk.clientSecret ) { %>
+        <link rel="stylesheet" href="https://unpkg.com/gitalk/dist/gitalk.css">
```

Comment:
I prefer to import css or js with `//` which can make protocol always suitable. Both effect with http and https. Don't you think so ?

Reply:
Good idea, I'll delete it.